### ISSUE TITLE
Currency Kit should format number

### ISSUE THREAD

**Issue by jasoncypret** [powerhome/playbook#331]
![image](https://user-images.githubusercontent.com/863031/66500571-622b4c00-ea87-11e9-8426-d9ca4e4994d6.png)

1. Dollar sign should probably be renamed to currency indicator (we might go international). Also, it should be the default.
2. The unit and period should be reworked.
    - Either pass a value with decimals and it splits it into the unit
    - Or when the unit is present add the period automatically
3. The comma should be added/formatted automatically
4. The React kit is not using the correct color for the unit.

**Comment by jasperfurniss** [comment]
My thought here is to utilize the power of an ActionView helper. 
https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_currency

I noticed we started doing that in some of our rails kits. 
Ex. phone.rb, we use the number_to_phone helper and it works perfect!
```
      def number
        if is_set? configured_number
          return number_to_phone(formatted_number, area_code: true) if formatted_number
        end
      end
```

This is the direction i'm headed down right now.
Wanted to check with you all before I go that route to see if there were any tips/thoughts.
@jasoncypret @thestephenmarshall @markiearnold @garettarrowood
